addons/l10n_mx/data/account_configuration.yml

```diff
@@ -0,0 +1,10 @@
+-
+  Account settings set for Use Anglo-Saxon Accounting
+-
+  !record {model: account.config.settings, id: mx_default_account_properties}:
+    {
+      'use_anglo_saxon': 1,
```

Comment:
Nhomar, you know our real case of a DB with localization for Mexico, Panama and Spain, so this auto-configuration is a real problem. Let's see what @qdp-odoo thinks about this.